lib/nostrum/struct/guild/role.ex

```diff
@@ -64,6 +65,10 @@ defmodule Nostrum.Struct.Guild.Role do
   @typedoc since: "0.7.0"
   @type icon :: String.t() | nil
 
+  @typedoc "The standard unicode character emoji icon for the role"
+  @typedoc since: "0.7.0"
```

Comment:
Not sure if I should be adding this in PR's so let me know (copied from above)

Reply:
This is good, thank you